webpack/src/Extends/Host/PuppetTab/SubTabs/Reports/index.js

```diff
@@ -28,6 +40,13 @@ const Reports = ({ hostName }) => (
 
 Reports.propTypes = {
   hostName: PropTypes.string.isRequired,
+  hostInfo: PropTypes.object,
```

Comment:
Note to myself, reports tab has moved to core which mean we can import it directly and remove slot&fill in this case, cc @nadjaheitmann @amirfefer